### PR TITLE
Issue18/add lic 11

### PR DESCRIPTION
**Pull request by angelicaengstrom**
My proposal for issue #18 regarding LIC 11, this includes

* functioanlity for LIC 11

* javadoc
* false and positive tests

Please review and suggest feedback

### CODE REVIEW

File: src/main/java/decide/LIC.java [hunk 1]
```diff
@@ -74,7 +99,7 @@ public boolean[] calculateCMV() {
         CMV[7] = LIC_7();
         CMV[8] = LIC_8();
         CMV[9] = LIC_9();
-        CMV[10] = LIC_10();
+        //CMV[10] = LIC_10();
```
Comment (tohanss) [inline src/main/java/decide/LIC.java:102]: Should not be commented out

**Review by tohanss — CHANGES_REQUESTED** [review]
Looks good, but similar to other PR with the comment.

**Review by TerenceNg03 — CHANGES_REQUESTED** [review]
Please merge CI from main first, so the check will run on this pr.

**Review by tohanss — APPROVED** [review]

File: src/main/java/decide/LIC.java (hunk 1 above)
Comment (TerenceNg03) [reply src/main/java/decide/LIC.java:102]: Would be fixed later.

**Pull request merged**